src/mlpack/methods/dbscan/random_point_selection.hpp

```diff
@@ -27,16 +27,24 @@ class RandomPointSelection
   /**
    * Select the next point to use, randomly.
    *
-   * @param unvisited Bitset indicating which points are unvisited.
-   * @param data Unused data.
+   * @param point Unused data.
+   * @param data Dataset to cluster.
    */
   template<typename MatType>
-  static size_t Select(const boost::dynamic_bitset<>& unvisited,
-                       const MatType& /* data */)
+  size_t Select(const size_t /* point */,
+                       const MatType& data /* data */)
```

Comment:
Another picky style comment, the second parameter should align with the first one. Also no need to write ``/* data */`` after the parameter, the comment above should be enough.

Reply:
Yes!! I will edit it